### PR TITLE
chore(color-wheel): use core tokens

### PR DESCRIPTION
**Pull request by jenndiaz**
## Description

Migrates ColorWheel to the core tokens system. 

Jira Ticket [CSS 189](https://jira.corp.adobe.com/browse/CSS-189)

## Related issue(s)

- Token Migration

## How has this been tested?

-   [ ] _Test case 1_
    1. Go here
    2. Do this
-   [ ] _Test case 2_
    1. Go here
    2. Do this

## Types of changes

-   [ ] Bug fix (non-breaking change which fixes an issue)
-   [ ] New feature (non-breaking change which adds functionality)
-   [ ] Breaking change (fix or feature that would cause existing functionality to change)
-   [ ] Chore (minor updates related to the tooling or maintenance of the repository, does not impact compiled assets)

## Checklist

-   [x] I have signed the [Adobe Open Source CLA](http://opensource.adobe.com/cla.html).
-   [ ] My code follows the code style of this project.
-   [ ] If my change required a change to the documentation, I have updated the documentation in this pull request.
-   [ ] I have read the **[CONTRIBUTING](<(https://github.com/adobe/spectrum-web-components/blob/main/CONTRIBUTING.md)>)** document.
-   [ ] I have added tests to cover my changes.
-   [ ] All new and existing tests passed.
-   [ ] I have reviewed at the Accessibility Practices for this feature, see: [Aria Practices](https://www.w3.org/TR/wai-aria-practices/)

## Best practices

This repository uses conventional commit syntax for each commit message; note that the GitHub UI does not use this by default so be cautious when accepting suggested changes. Avoid the "Update branch" button on the pull request and opt instead for rebasing your branch against `main`.

### CODE REVIEW

**Comment by jenndiaz** [comment]
All VRT failures as expect as new designs required significant changes to the size and appearance of the component. 

Spectrum-CSS has merged and graduated to the full release.

**Comment by Westbrook** [comment]
@hunterloftis @najikahalsema I retargeted this PR to `lightningcss` so that we could take advantage of the new config and not need to refactor the work a second time. With the updated config and JS, would yall take a look at this?

File: packages/color-wheel/src/spectrum-config.v2.js [hunk 1]
```diff
@@ -24,6 +27,10 @@ const config = {
             inPackage: '@spectrum-css/colorwheel',
             outPackage: 'color-wheel',
             fileName: 'color-wheel',
+            excludeByComponents: [
+                builder.class('🤫'),
```
Comment (hunterloftis) [inline packages/color-wheel/src/spectrum-config.v2.js:31]: ?

**Review by hunterloftis — APPROVED** [review]

File: packages/color-wheel/src/spectrum-config.v2.js (hunk 1 above)
Comment (Westbrook) [reply packages/color-wheel/src/spectrum-config.v2.js:31]: CSS needs this to not make their tooling angry and then it ends up in our CSS if we don't exclude it 😞.

**Pull request merged**